Sender_BMS/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Threading;
+
+namespace Sender_BMS
+{
+    public class Program
+    {
+        private static bool cancelStream = false;
+
+        public static void Main(string[] args)
+        {
+            Console.WriteLine("Battery parameters streaming has started. Press Ctrl-C to stop streaming");
```

Comment:
Your console output is intended to be parsed by another 'receiver' program. Did you consider the Receiver complexity to parse or ignore English strings in your output?